work_items/WACI_presentation_exchange.md

```diff
@@ -0,0 +1,64 @@
+# WACI Presentation Exchange
+
+
+## Work Item Owners
+- Orie Steele (@OR13)
+- Brent Zundel (@brentzundel)
+
+## Outcome
+**When your work item is successful, then what will have changed in the world? What
+will we be able to do that wasn't possible do before or vice versa?**
+
+We will have a standard protocol for requesting and submitting a presentation of
+a variety of credentials formats. 
+
+## Deliverables
+- **Does your work item have any deliverables? These include code projects,
+  standards-track specifications, reports, registries, summaries, blog posts,
+  and anything else that is instantiable and publishable in written form to a
+  broader audience.**
+
+We are writing a standards-track specification.
```

Comment:
Should we specifically mention test-vectors or interop profiles here or are these covered via https://github.com/decentralized-identity/org/blob/master/Org%20documents/Membership%20agreements/Project%20Charter%20Addendum.md

Reply:
Proposed: https://github.com/decentralized-identity/claims-credentials/pull/5/files#r620548576